pkg/csi/service/wcp/controller.go

```diff
@@ -148,6 +147,19 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 						time.Sleep(5 * time.Second)
 					}
 				}
+				if event.Op&fsnotify.Create == fsnotify.Create {
```

Comment:
Can we also update the condition above `event.Op&fsnotify.Remove == fsnotify.Remove` and make sure it does not get invoked when the certificate is rotated.

Reply:
fsnotify.Create event only

Reply:
> Can we also update the condition above event.Op&fsnotify.Remove == fsnotify.Remove and make sure it does not get invoked when the certificate is rotated.

Added